modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -50,7 +49,46 @@ Some actions are not yet supported:
 // We are waiting for SMT to release the feature/fix to mirror Debian repositories. When this has been done, this comment and the limitation above can be removed.
 
 
-This section describes how to manually prepare {ubuntu} clients for registration to your {susemgr} Server and synchronize the repositories.
+This section describes how to synchronize the repositories and prepare {ubuntu} clients for registration to your {susemgr} Server.
+
+
+== {ubuntu} Channel Management Tips
+
+This section provides tips on {ubuntu} channel management.
+
+* The base parent distribution {ubuntu} channel contains zero packages. No base media is provided by {suse}. The {ubuntu} media should be added as child channels of the {ubuntu} parent channel.
```

Comment:
The {ubuntu} base channel does not contain any packages, and {suse} does not provide any base packages.